app/views/dishes/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="page-header">
-  <h1>
-    Dishes
-    <a href="/dishes/new" class="btn btn-lg btn-success">New Dish</a>
-    <a href="/venues/new" class="btn btn-lg btn-success">New Venue</a>
-  </h1>
+  <!--<h1>-->
```

Comment:
Can you just delete the commented out HTML? That way, it's cleaner -- and we have the old version stored in git anyway, so we can revert easily.